public/maps/map.js

```diff
@@ -0,0 +1,189 @@
+/**
```

Comment:
This is refactored from https://github.com/art-institute-of-chicago/aic-mobile-cms/blob/3fab295bd5f059b4046ec13910c87758a9097148/sites/all/modules/custom/aicapp/map-page.php